fpdf/fpdf.py

```diff
@@ -4998,6 +4998,7 @@ def table(self, *args, **kwargs):
             num_heading_rows (number): optional. Sets the number of heading rows, default value is 1. If this value is not 1,
                 first_row_as_headings needs to be True if num_heading_rows>1 and False if num_heading_rows=0. For backwards compatibility,
                 first_row_as_headings is used in case num_heading_rows is 1.
+            repeat_headings (bool): optional, indicates whether to print table headings on every page, default to True.
```

Comment:
We have a number of possible heading placement scenarios here:

* start of table
* top of page after a page break between rows
* top of page after a page break in the middle of a row
* any others?

Not sure if it would be desirable to distinguish between those. But if it is, then an enum might be more future proof here than a bool.

Reply:
Agreed